src/sparseml/export/helpers.py

```diff
@@ -54,6 +56,25 @@ def format_source_path(source_path: Union[Path, str]) -> str:
     return source_path.as_posix()
 
 
+def onnx_data_files(onnx_data_name: str, path: Union[str, Path]) -> List[str]:
+    """
+    Given the onnx_data_name, return a list of all the onnx data file names
+    in the src_path. E.g. if onnx_data_name is "model.data", return
+    ["model.data"] (if there is only one file present),
+    alternativelty potentially return ["model.data.0", "model.data.1", ...]
+    if the files are split into multiple files.
+
+    :param onnx_data_name: The name of the onnx data file.
+    :param path: The path to the onnx data file.
+    :return: A list of all the onnx data file names.
+    """
+    onnx_data_pattern = re.compile(rf"{onnx_data_name}(\.\d+)?$")
```

Comment:
probably a dumb question, but is this format of data files guaranteed for any onnx model or is this our own convention?

Reply:
And nit: but maybe we should ensure "data" as a string is in the regex

Reply:
It is a convention used by us and HF optimum. But there is no guarantee everyone will use the same convention.